core/embed/rust/src/ui/macros.rs

```diff
@@ -4,3 +4,35 @@ macro_rules! include_res {
         include_bytes!(concat!(env!("CARGO_MANIFEST_DIR"), "/src/ui/", $filename))
     };
 }
+
+#[allow(unused_macros)] // Only used in TR so far.
+/// Concatenates arbitrary amount of slices into a String.
+macro_rules! build_string {
```

Comment:
The benefit of this being a macro instead of a function is that one can use it without any import. Not sure what are the implications like binary size etc. Does it have any advantage of using a function instead?